### ISSUE TITLE
Feature Request: Video Call

### ISSUE THREAD

**Issue by carmat88** [threema-ch/threema-web#614]
Hello Folks! It goes without saying that a project such Threema is like a little safe island where getting some fresh air in an ocean of D(ata)sharks. Therefore, thank you so much for your work.

Nevertheless and funky analogies aside, I am honestly puzzled and disappointed about not seeing any sign of interest/development from both the company and the community regarding two kinds of "basic/helpful" feature in nowadays application for instant messaging. Both _Wire_ and _Signal_ provide them already, so as mentioned I am really puzzled. Why Threema is not investing time and resources in such certainly non-trivial technical challenge?

In conclusion, like it has been already proposed almost one year ago by @joergister in https://github.com/threema-ch/threema-web/issues/370, it would be really great, helpful and much appreciated to have also the feature of video calls in Threema (maybe easier to be implemented in the web GUI ? Frankly I have no clue for the time being) in addition to call someone via threema-web.

Hope this would be considered for future implementations.

Thanks again for your dedication to such project.

Have a good one!

**Comment by dbrgn** [comment]
Hi. This (as well as #370) is definitely on our TODO-List :slightly_smiling_face: 

Since the feature request is mainly about the apps, I'll close this issue for now (we try to keep the issue tracker specific to the Threema Web frontend). But once the apps have video call support, #370 would also need to include that functionality of course.

**Comment by carmat88** [comment]
Hey @dbrgn ! Thanks for getting back so quickly. However as a user and member of the community I do wonder how one should read the above comment timewise :-) Any loose estimation?
> This (as well as #370) is definitely on our TODO-List 

Cheers.

**Comment by dbrgn** [comment]
No, right now we can't really estimate when it will be ready :slightly_smiling_face: Unless "probably in less than a year" is of any help. But company policy is "it's ready when it's ready" (people don't like broken promises).

**Comment by carmat88** [comment]
Howdy again @dbrgn ! Thanks for clarifying and expanding the above reply, much appreciated. As you guys are acknowledging and probably are aware of, these are features that will arguably boost the interest and trustworthiness of the Threema project in general; to not mention that you will alleviate the almost "existential" dilemma of many users out there when using the plague of opaque, greedy technologies and thus become even more reliable and competitive within the market. With that being said, I do see the other side of the coin; in other words, the company's perspective and thus policy of "_it's ready when it's ready_" since people don't like broken promises indeed (and I would add non-stable releases as well). In any event, please keep up with the great work and do not give up pressing the upper management regarding the aforementioned features ;-) Tschüss!

**Comment by lgrahl** [comment]
> [...] pressing the upper management [...]

:thinking: That's not a management issue. It's a matter of (wo)man hours. :)
